outlines/welcome.md

```diff
@@ -0,0 +1,30 @@
+1. Welcome to the Meteor guide
+  1. This is a set of articles outlining opinions on best-practice application development in the Meteor framework (and in many cases other frameworks)
+  2. Nothing here is *required* (i.e. you can use the platform in many other ways), but this is how we think apps are best developed. Take what you want from it but be aware that these are the community conventions
+  3. Hard APIs can be found at [docs] and community packages at [atmosphere].
+  4. Getting started with Meteor? Use the learning docs below *first*, come back later.
+2. What is Meteor?
+  1. A set of JS client + server code for building web and mobile applications, built and curated by MDG and the Meteor community.
+  2. A build tool for building JS bundles for deployment
```

Comment:
3. A fantastic package ecosystem that includes packages built to work specifically with Meteor in a client/server environment as well as access to the full spectrum of packages in npm.